### PR TITLE
Docs review

### PR DESCRIPTION
**Pull request by Blackbaud-LindseyRix**
- Casing updates
- Add more instances of restricted view alerts
- Switch Stache action buttons to SKY action buttons
- Add initial draft of manual routing documentation
- Remove changelog pages that are no longer needed
- Include previous search docs enhancements that never went live
- Run `skyux upgrade`

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/stache?_sp.spa=pr-docs-review](https://host.nxt.blackbaud.com/stache?_sp.spa=pr-docs-review)

File: src/app/components/search/index.html [hunk 1]
```diff
@@ -274,5 +274,40 @@ <h3>
...
+    If you have tokens or sensitive information on the page, wrap that section in the <sky-code>&lt;stache-hide-from-search></sky-code> component.
+  </li>
+  <li>
+    If you have test specs, you may need to adjust them for the search.
+  </li>
+  <li>
+    Run <sky-code>skyux serve</sky-code> to ensure that the pages still appear correctly.
+  </li>
+  <li>
+    If everything is working and looking good, add, commit, and push up your branch changes to the repo and create a pull request into the master branch.
```
Comment (blackbaud-paulah) [inline src/app/components/search/index.html:304]:
```suggestion
    If everything is working and looking good, create a pull request to merge your branch changes into the master branch.
```

File: src/app/components/search/index.html [hunk 2]
```diff
@@ -274,5 +274,40 @@ <h3>
...
+  </li>
+</ol>
+
+<p>
+  Once you complete these steps and merge this branch to master, the new pages will be scraped by the search and available to query. Unfortunately, the scraping doesn't occur until release so you won't be able to test this locally first.
```
Comment (blackbaud-paulah) [inline src/app/components/search/index.html:309]:
```suggestion
  After you complete these steps and merge this branch to master, the new pages will be scraped by the search and available to query. Unfortunately, the scraping doesn't occur until release so you won't be able to test this locally first.
```

File: src/app/components/search/index.html [hunk 3]
```diff
@@ -274,5 +274,40 @@ <h3>
...
+</p>
+<p>One additional item to consider... if you don't want to have Stache on a tooling page, just reconsider how you set up your site. Can you add Stache to a different documentation only page? Or, can you check how discoverable this page is? There may be other ways to get the content you need to show up in search if a particular page isn't well suited towards adding Stache.</p>
```
Comment (blackbaud-paulah) [inline src/app/components/search/index.html:311]:
```suggestion
<p>One additional item to consider... if you don't want to have Stache on a tooling page, reconsider how you set up your site. Can you add Stache to a different documentation only page? Or, can you check how discoverable this page is? There may be other ways to get the content you need to show up in search if a particular page isn't well suited towards adding Stache.</p>
```

**Review by blackbaud-paulah — APPROVED** [review]

**Review by blackbaud-paulah — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/stache?_sp.spa=pr-docs-review](https://host.nxt.blackbaud.com/stache?_sp.spa=pr-docs-review)

**Pull request merged**